src/tree/mod.rs

```diff
@@ -16,8 +16,6 @@ pub use self::frag::*;
 pub use self::data::*;
 pub use self::node::*;
 
-pub const FANOUT: usize = 2;
```

Comment:
this was a huge facepalm, we were basically hardcoded to a binary tree lol

Reply:
Lol! That's a good one! 😄